README.md

```diff
@@ -288,6 +288,30 @@ OpenMock leverages [https://golang.org/pkg/text/template/](https://golang.org/pk
   {{.HTTPBody | xmlPath "node1/node2/node3"}}
   ```
 
+# Admin Interface
+Openmock also opens an admin interface on port 9998, which allows you to see and 
+manipulate the loaded templates.
```

Comment:
the word 'template' is getting increasingly overloaded in openmock. should we start referring to these as behaviors/behavior definitions?

Reply:
Hm, I definitely agree about the overloading. This interface provides CRUD for om's entire model (templates, behaviors, abstract behaviors, etc), so I'm not sure the best terminology.  I like 'definitions'.